data/pack/install.yml

```diff
@@ -0,0 +1,79 @@
+- name: Linux
+  methods:
+    - name: Arch Linux
+      content: |
+        On Arch Linux, there are two packages available:
+        
+        - [pack-cli](https://aur.archlinux.org/packages/pack-cli/)
+        - [pack-cli-bin](https://aur.archlinux.org/packages/pack-cli-bin/)
+    - name: Command
+      content: |
+        The following command will download the latest version of `pack` from [GitHub](https://github.com/buildpacks/pack/releases) and install it in `/usr/local/bin/`.
+
+        ```bash
+        (curl -sSL "https://github.com/buildpacks/pack/releases/download/v<PACK_VERSION>/pack-v<PACK_VERSION>-linux.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack)
+        ```
+    - &container
+      name: Container
+      content: |
+        `pack` is also available as container images on Docker Hub as [`buildpacksio/pack`](https://hub.docker.com/r/buildpacksio/pack)
+        ([definition files](https://github.com/buildpacks/pack/blob/main/.github/workflows/delivery/docker/Dockerfile)).
+
+        #### Tags
+
+        * Use this tag to track the latest release:
+            * `buildpacksio/pack:latest`
+        * Use a version tag to pin a specific release:
+            * `buildpacksio/pack:<PACK_VERSION>`
+            * [other versions](https://hub.docker.com/r/buildpacksio/pack/tags)
+
+        #### Usage
+
+        In some container environments you may be required to mount your local Docker daemon's socket.
+
+        For example, using the Docker CLI:
+
+        ```shell
+        docker run \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          -v $PWD:/workspace -w /workspace \
+          buildpacksio/pack build <my-image> --builder <builder-image>
+        ```
+    - &homebrew
+      name: Homebrew
+      default: true
+      content: |
+        `pack` can be installed via [Homebrew](https://brew.sh/):
+        
+        ```bash
+        brew install buildpacks/tap/pack
+        ```
+    - &manually
+      name: Manually
+      content: |
+        To manually install `pack`, visit the [releases page on GitHub](https://github.com/buildpacks/pack/releases). 
+        1. Download the `.tgz` or `.zip` file for your platform
+        2. Extract the `pack` binary
+        3. (Optional) Add the directory containing `pack` to `PATH`
+- name: macOS
+  default: true
+  methods:
+    - *container
+    - *homebrew
+    - *manually
+- name: Windows
```

Comment:
I would order it as:
```
* Chocolatey
* Scoop
* Container
* Manually
```

So that both CLI install methods are next to each other (because anyways, the way it is now, it's not alphabetical). What do you think?

Reply:
I choose to keep the tabs alphabetically ordered as to not be bias towards any tool although we kind of are when we default to "homebrew". Open to discussion but I prefer alphabetical when looking at various options.